keras/utils/conv_utils.py

```diff
@@ -49,7 +49,7 @@ def convert_data_format(data_format, ndim):
         'Expected values are ["channels_first", "channels_last"]')
 
 
-def normalize_tuple(value, n, name):
+def normalize_tuple(value, n, name, validation_fn=lambda x: x > 0, fn_alt='>0'):
```

Comment:
errr, this method has way too many param now.

Reply:
Done